App.js

```diff
@@ -11,7 +11,7 @@ import { Linking } from 'react-native'
 
 import store from './app/store'
 
-import Router from './app/router'
+import Router from './app/Router'
```

Comment:
plz also remove this change

Reply:
Removed but it throws a bundler error

![Screenshot 2020-10-12 at 09 53 14](https://user-images.githubusercontent.com/805569/95714456-c8791a80-0c57-11eb-98e7-00bb1720dbef.png)

Reply:
for me your changes crashes - i think its because you have already changed folder name and then revert commit folder name back - but git is bad with letter cases changes on windows - we was facing problems like this and thats the reason i`m not changing folders/files cases now